src/redshift-adjust

```diff
@@ -42,10 +42,14 @@ EOF
 fi
 
 if [ ! -e "${file_temp}" ]; then
-    echo 6500 > "${file_temp}"
+    temperature=$(redshift -p | grep -Pzo "(?<=Color temperature: ).*")
+    temperature=${temperature%?} # remove last char (K)
+    echo $temperature > "${file_temp}"
 fi
 if [ ! -e "${file_bright}" ]; then
-    echo 1.0 > "${file_bright}"
+    brightness=$(redshift -p | grep -Pzo "(?<=Brightness: ).*")
+    echo $brightness > "${file_bright}"
+    $(pkill redshift) # killing redshift prevents default config to override new config
```

Comment:
This should just be `pkill '^redshift$'`.

Reply:
Thanks, your suggestion solved a bug I was experiencing that the screen would blink and the settings would not be applied in the first time I was running redshift-adjust and redshift was running in the background.